src/utils/formatting.ts

```diff
@@ -99,3 +99,18 @@ export const formatTimeToHMS = (time: string) => {
     seconds
   )}s`;
 };
+
+export const formatMarketId = (marketId: string) => {
+  const timestamp = marketId.slice(0, 6);
+  const location = marketId.slice(6, 9);
+  const place = marketId.slice(9, 11);
```

Comment:
you can use `marketId.slice(-2)` to get the last two characters here, it's also the race number not the place - might be confusing if someone isn't familiar with the `marketId` format